package.json

```diff
@@ -11,7 +11,7 @@
     "blockchain:contracts": "node --experimental-modules ./tests/e2e/setup-contracts.mjs",
     "build": "run-s check-types build:css build:app",
     "build:dev": "cross-env NODE_ENV=development npm run build",
-    "build:test": "cross-env NODE_ENV=test NET=ropsten npm run build",
+    "build:test": "cross-env NODE_ENV=test NET=goerli npm run build",
```

Comment:
Just for info, IIRC, the `NET=goerli` is optional because goerli is already the default network. But no harm leaving it here.

Reply:
yup, i think will keep it there. just to be abit more explicit